### ISSUE TITLE
traceback on skipped test

### ISSUE THREAD

**Issue by sagzest** [Tbruno25/pytest-timestamps#4]
This is a great pluggin, but, unfortunately, I'm encountering a traceback on a test decorated with `pytest.mark.skipif`:
```
path/to/my/tests/test_integration.py::test_my_test SKIPPED (Not running weekly tests)
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/_pytest/main.py", line 270, in wrap_session
INTERNALERROR>     session.exitstatus = doit(config, session) or 0
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/_pytest/main.py", line 324, in _main
INTERNALERROR>     config.hook.pytest_runtestloop(session=session)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_hooks.py", line 433, in __call__
INTERNALERROR>     return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_manager.py", line 112, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_callers.py", line 155, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_result.py", line 108, in get_result
INTERNALERROR>     raise exc.with_traceback(exc.__traceback__)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_callers.py", line 80, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/_pytest/main.py", line 349, in pytest_runtestloop
INTERNALERROR>     item.config.hook.pytest_runtest_protocol(item=item, nextitem=nextitem)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_hooks.py", line 433, in __call__
INTERNALERROR>     return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_manager.py", line 112, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_callers.py", line 155, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_result.py", line 108, in get_result
INTERNALERROR>     raise exc.with_traceback(exc.__traceback__)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_callers.py", line 80, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/_pytest/runner.py", line 114, in pytest_runtest_protocol
INTERNALERROR>     runtestprotocol(item, nextitem=nextitem)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/_pytest/runner.py", line 127, in runtestprotocol
INTERNALERROR>     rep = call_and_report(item, "setup", log)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/_pytest/runner.py", line 226, in call_and_report
INTERNALERROR>     hook.pytest_runtest_logreport(report=report)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_hooks.py", line 433, in __call__
INTERNALERROR>     return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_manager.py", line 112, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_callers.py", line 116, in _multicall
INTERNALERROR>     raise exception.with_traceback(exception.__traceback__)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pluggy/_callers.py", line 80, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pytest_timestamps/plugin.py", line 46, in pytest_runtest_logreport
INTERNALERROR>     super().pytest_runtest_logreport(report)
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/_pytest/terminal.py", line 622, in pytest_runtest_logreport
INTERNALERROR>     self._write_progress_information_filling_space()
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pytest_timestamps/plugin.py", line 35, in _write_progress_information_filling_space
INTERNALERROR>     self._write_ts_to_terminal()
INTERNALERROR>   File "/Users/someone/.pyenv/versions/my_env/lib/python3.10/site-packages/pytest_timestamps/plugin.py", line 28, in _write_ts_to_terminal
INTERNALERROR>     start, stop = self._get_timestamps()
INTERNALERROR> ValueError: not enough values to unpack (expected 2, got 0)
```

In case the info is needed, I'm using the pluggins,
```
plugins: syrupy-3.0.6, remotedata-0.4.0, timestamps-0.1.4, nbval-0.10.0, cov-4.1.0, anyio-3.7.0
```

**Comment by Tbruno25** [comment]
Thanks for reporting the issue.

You're correct -- it seems to error out when any `skip` decorator is used. 

Will look into a fix :+1:

**Comment by Tbruno25** [comment]
Published version `0.2.0` to pypi which should resolve the issue :slightly_smiling_face:
